packages/ragbits-core/src/ragbits/core/cli.py

```diff
@@ -1,21 +1,52 @@
 # pylint: disable=import-outside-toplevel
+# pylint: disable=missing-param-doc
+from pathlib import Path
+
 import typer
 
+from ragbits.core.config import core_config
+
 prompts_app = typer.Typer(no_args_is_help=True)
 
 
-def register(app: typer.Typer, help_only: bool) -> None:
+def register(app: typer.Typer) -> None:
     """
     Register the CLI commands for the package.
 
     Args:
         app: The Typer object to register the commands with.
-        help_only: A boolean indicating whether it is a help-only run.
     """
-    if not help_only:
-        from .prompt.lab.app import lab_app
-        from .prompt.promptfoo import generate_configs
 
-        prompts_app.command(name="lab")(lab_app)
-        prompts_app.command(name="generate-promptfoo-configs")(generate_configs)
+    @prompts_app.command()
+    def lab(
+        file_pattern: str = core_config.prompt_path_pattern,
+        llm_factory: str | None = core_config.default_llm_factory,
+    ) -> None:
+        """
+        Launches the interactive application for listing, rendering, and testing prompts
+        defined within the current project.
+        """
+        from ragbits.core.prompt.lab.app import lab_app
+
+        lab_app(file_pattern=file_pattern, llm_factory=llm_factory)
+
+    @prompts_app.command()
+    def generate_promptfoo_configs(
+        file_pattern: str = core_config.prompt_path_pattern,
+        root_path: Path = Path.cwd(),
+        target_path: Path = Path("promptfooconfigs"),
+    ) -> None:
+        """
+        Generates the configuration files for the PromptFoo prompts.
+
+        This function registers all the CLI modules in the ragbits packages:
+            - iterates over every package in the ragbits.* namespace
+            - it looks for `cli` package / module
+            - if found it imports the `register` function from the `cli` module and calls it with the `app` object
+            - register function should add the CLI commands to the `app` object
```

Comment:
It seems like a docstring from a different function

Reply:
thx fixed